src/components/ThemeSwitch.tsx

```diff
@@ -1,13 +1,17 @@
 import { FC } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
+import { WebviewBackground } from 'webview-background'
 import { settingsActionCreator as setting } from '../actions/settings'
 import theme from '../selectors/theme'
+import themeColors from '../selectors/themeColors'
 import Checkbox from './Checkbox'
 
 /** A ThemeSwitch Component with a title and description. */
 const ThemeSwitch: FC = () => {
   const dispatch = useDispatch()
   const light = useSelector(state => theme(state) === 'Light')
+  const colors = useSelector(themeColors)
+  WebviewBackground.changeBackgroundColor({ color: colors.bg })
```

Comment:
Similarly, we might want to put this into a `useEffect` hook so that it is only called when the color changes.

Reply:
Placed inside a `useEffect` hook also: 25268172daa0c7e55f005a587e9c803a5f2293d3